src/Icon/Icon.js

```diff
@@ -31,14 +31,17 @@ export default class Icon extends React.Component {
   };
 
   getLocals() {
+    const { className, icon, color, style = {} } = this.props;
```

Comment:
@FrancescoCioria suggested this alternative:
```
getLocals({ className, icon, color, style = {} }) {}
```

OTOH, The Book:tm: [does not agree](https://buildo.gitbooks.io/book/content/frontend/tutorial/step_3.html).